src/common/libsubprocess/remote.c

```diff
@@ -673,6 +561,8 @@ static void rexec_continuation (flux_future_t *f, void *arg)
     if (subprocess_rexec_is_started (f, &p->pid)) {
         p->pid_set = true;
         process_new_state (p, FLUX_SUBPROCESS_RUNNING);
+        if (send_channel_data (p) < 0)
+            goto error;
```

Comment:
Should we destroy the fbuf here?

Reply:
ahhh you mean in the error case if it wasn't destroyed in `send_channel_data()`.  Hmmm, we certainly could.  But since we're `goto error`, we're failing the subprocess and going to destroy the subprocess, so shortly.  So not super necessary?

Reply:
Oh, derp!  I didn't notice that it was destroyed in `send_channel_data()`.  NM!